### PR TITLE
Update and rename comments.md to comments-in-python.md

### PR DESCRIPTION
**Pull request by sammillersmith**

### CODE REVIEW

**Comment by sammillersmith** [comment]
I renamed the file, and so now the diff is just all green...

**Comment by sammillersmith** [comment]
In any case, please skim through, accept or reject any and all edits, and then feel free to publish.

File: content/python/encyclopedia/comments-in-python.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Instead, multiple `#` characters can be used:
+
+```py
+# This is a comment written over
+# more than one line
+
+print("Hello, World!")
+```
+
+Another, less official way of writing comments in Python is to use a multi-line string. Python will ignore string literals that are not assigned to a variable, so multi-line strings (created by surrounding text with triple quotes `"""`) can be used as de facto comments:
```
Comment (sonnynomnom) [inline content/python/encyclopedia/comments-in-python.md:39]: like this!

**Review by sonnynomnom — APPROVED** [review]
lgtm

**Comment by sonnynomnom** [comment]
> I renamed the file, and so now the diff is just all green...

noted! (same name as title... i forgot here)

**Comment by sonnynomnom** [comment]
publishing!

**Pull request merged**